content/soarian/dstu2.md

```diff
@@ -0,0 +1,69 @@
+The Application interacts with the API using OAuth2 and the IdP with SAML for trusted user connections.  When the patient is seen for a healthcare encounter, Soarian Clinicals establishes a temporary linking token which is provided to the patient. Individual tokens may also be assigned to patient-authorized proxy(ies). The Soarian Clinicals user is responsible for state and federal guidelines for interacting with the patient and managing proxies as entered in Soarian. The Application must provide a means for the Application user to establish themselves in an approved identity server and then enter the temporary linking token into the Application. The Application interacts with an authorization server to establish a trusted connection and user authentication and then passes the temporary linking token to the Soarian Clinicals ActivateLinkToken API and Soarian Clinicals establishes a user-patient relationship that is used in subsequent interactions. This interaction provides the Application a trusted link between the user definition used for authentication and the identity of that user as a Soarian managed patient or proxy thus authorizing each user for a specific patient's data. Each subsequent interaction between the Application and API utilizes the trusted connection and a patient scope selection that Soarian has authorized via the token exchange.
+
+The below figure shows the interaction summary between Application, IdP, Authorization server and Soarian API. 
+
+![Soarian API and Application interaction](/images/soarian-api-application-interaction.jpg)
+
+Once the IdP, Application, and API are registered, API is enabled and the user-patient-proxy identity and authorization is established, the patient/proxy Application user can use Application capabilities to query the API for:
+ * One or more C-CDA SOEN (consolidated clinical documentation architecture summary of episode note) containing an authorized patient's health information 
+ * Each category of health data such as a medication list, allergy list, problem list, diagnostic test results, or other specified information.  
+
+The Application is responsible for determining which of and how the API capabilities are exposed to the user. For each capability the Application offers the user, the FHIR resource pages define those API capabilities and, within each, which capabilities the Application must/shall support to be compliant with the API.
+
+## Special information regarding date parameters
+
+* All date formats are ISO 8601 formatted: YYYYMMDD.
+
+* Date is assumed formatted properly and always evaluated left to right: YYYYMMDD.  Note specifically that if 20152101 is used, MM is 21 or 12+9 months and the API will resove 20152101 to 20160901.
```

Comment:
"Dates are assumed to be formatted correctly. The API does not reject dates with out of bounds months or days. Instead, the API resolves them to valid amounts. For example, if a date of 20152101 is provided, MM is 21 or 12+9 months and the API will resolve 20152101 to 20160901."

"resove" -> "resolve" regardless of any other changes here